libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/cert/CertRecordStoreConnection.java

```diff
@@ -73,8 +73,16 @@
     int deleteExpiredX509CertRecords(int expiryTimeMins);
 
     /**
-     * List all certificates that failed to refresh for more than three days.
-     * @param lastNotifiedServer
+     * Return all certificate records that failed to refresh after updating them with the current notification time and server.
+     * Implementers of this interface should note the following:
+     * 1. The records should be filtered according to the following criteria:
+     *      - The certificates should have a grace period in which no notifications will be sent. The default grace period is 2 weeks
```

Comment:
I mentioned the default grace period is 2 weeks as this will be the case after the "grace period" PR will be merged